packages/oid4vci-demo-frontend/src/ecosystem/ecosystem-config.ts

```diff
@@ -109,9 +109,19 @@ export interface SSIWelcomePageConfig extends PageConfig {
 
 export interface SSICredentialIssuedSuccessPageConfig extends PageConfig {
     leftPaneWidth?: string
-    photoLeft: string
+    backgroundColor?: string
+    logo?: ImageProperties
+    photoLeft?: string
     photoRight: string
     rightPaneButtonStepId?: string
+    rightPaneTitle: string
```

Comment:
should all of this not be optional? how does this work with other demo's?

Reply:
Fixed